lib/taskana-core/src/test/java/acceptance/priorityservice/PriorityServiceAccTest.java

```diff
@@ -0,0 +1,97 @@
+package acceptance.priorityservice;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import acceptance.AbstractAccTest;
+import java.sql.Date;
+import java.time.Instant;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DynamicTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestFactory;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.function.ThrowingConsumer;
+
+import pro.taskana.classification.api.ClassificationService;
+import pro.taskana.classification.api.models.Classification;
+import pro.taskana.common.api.ScheduledJob;
+import pro.taskana.common.internal.util.Pair;
+import pro.taskana.common.test.security.JaasExtension;
+import pro.taskana.common.test.security.WithAccessId;
+import pro.taskana.task.api.TaskCustomField;
+import pro.taskana.task.api.TaskService;
+import pro.taskana.task.api.models.ObjectReference;
+import pro.taskana.task.api.models.Task;
+
+/** Acceptance test for all priority computation scenarios. */
+@Disabled("Until we enable the use of Test-SPI's in only specific tests")
+@ExtendWith(JaasExtension.class)
+class PriorityServiceAccTest extends AbstractAccTest {
+
+  private static final TaskService TASK_SERVICE = taskanaEngine.getTaskService();
+
+  @WithAccessId(user = "user-1-1")
+  @TestFactory
+  Stream<DynamicTest> should_SetThePriorityAccordingToTestProvider_When_CreatingTask() {
+    List<Pair<String, Integer>> testCases = List.of(Pair.of("false", 1), Pair.of("true", 10));
+
+    ThrowingConsumer<Pair<String, Integer>> test =
+        x -> {
+          Task task = TASK_SERVICE.newTask("USER-1-1", "DOMAIN_A");
+          task.setCustomAttribute(TaskCustomField.CUSTOM_6, x.getLeft());
+          task.setClassificationKey("T2100");
+          ObjectReference objectReference =
+              createObjectReference("COMPANY_A", "SYSTEM_A", "INSTANCE_A", "VNR", "1234567");
+          task.setPrimaryObjRef(objectReference);
+
+          Task createdTask = TASK_SERVICE.createTask(task);
+          assertThat(createdTask.getPriority()).isEqualTo(x.getRight());
+        };
+
+    return DynamicTest.stream(testCases.iterator(), x -> "entry in custom6: " + x.getLeft(), test);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @TestFactory
+  Stream<DynamicTest> should_SetThePriorityAccordingToTestProvider_When_UpdatingTask()
+      throws Exception {
+    List<Pair<String, Integer>> testCases = List.of(Pair.of("false", 1), Pair.of("true", 10));
+    Task task = TASK_SERVICE.getTask("TKI:000000000000000000000000000000000000");
+    int daysSinceCreated =
+        Math.toIntExact(
+            TimeUnit.DAYS.convert(
+                Date.from(Instant.now()).getTime() - Date.from(task.getCreated()).getTime(),
+                TimeUnit.MILLISECONDS));
```

Comment:
Why not use `Duration#between`? I think that would make it more readable.

Reply:
You're right. I'll change it.

Reply:
I realized that `Duration#between()` is providing a Duration, for which it is not possible to retrieve the days of it (only seconds & nanos). To compute the days it would require some additional not less complicated steps, as far as I see it.
But you're right that it would be easier to create a straight forward test implementation for reasons of comprehension.